--- techne-0.2/src/techne.in~	2010-09-05 22:36:14.000000000 +0200
+++ techne-0.2/src/techne.in	2010-12-14 12:01:45.414521334 +0100
@@ -2,7 +2,7 @@
 
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+bindir=@bindir@
 datarootdir=@datarootdir@
 datadir=@datadir@
 debug=""
@@ -23,4 +23,4 @@
     shift
 done
 
-${debug} ${libdir}/techne/techne.bin -Oprefix=${datadir}/techne $args
+${debug} ${bindir}/techne/techne.bin -Oprefix=${datadir}/techne $args
